MetafileImporter/Logic/vtkSlicerMetafileImporterLogic.cxx

```diff
@@ -421,6 +434,85 @@ vtkMRMLSequenceBrowserNode* vtkSlicerMetafileImporterLogic::ReadSequenceMetafile
   return sequenceBrowserNode.GetPointer();
 }
 
+//----------------------------------------------------------------------------
+vtkMRMLSequenceBrowserNode* vtkSlicerMetafileImporterLogic::ReadSequenceNRRD(const std::string& fileName, vtkCollection* addedSequenceNodes/*=NULL*/)
```

Comment:
This looks VERY similar to metaimage reading code. It would make this class shorter and easier to maintain if one method would handle both metaimage and nrrd file - similarly how you did it for ReadSequenceMetafileTransforms.